data/result-exporter/swe-cne-exporter/src/test/resources/SweCNEWithFailure_Z01.xml

```diff
@@ -150,7 +150,7 @@
                         <mRID>ac1</mRID>
                         <businessType>B87</businessType>
                         <name>ac1-name</name>
-                        <quantity.quantity>4.0</quantity.quantity>
+                        <quantity.quantity>4.00</quantity.quantity>
```

Comment:
did you just modify these files without regenerating RaoResults ?

Reply:
nope, I just increased the default number of decimals for CNE files to 2 to align with RaoResult

Reply:
after discussion we will rollback to 1 default decimal for CNE export since it could be a business requirement